src/v3/src/img/yubikeyDemo.svg

```diff
@@ -0,0 +1,19 @@
+<svg width="91" height="85" viewBox="0 0 91 85" fill="none" xmlns="http://www.w3.org/2000/svg" aria-labelledby="yubikeyDemo">
+<title id="yubikeyDemo">yubikey</title>
+<g clip-path="url(#clip0_201_13130)">
+<path d="M11.327 62.2236C8.20281 59.0994 8.20281 54.0341 11.327 50.9099L53.9891 8.24781L61.237 15.4957L75.7326 29.9913L82.9805 37.2392L40.3184 79.9013C37.1942 83.0255 32.1289 83.0255 29.0047 79.9013L11.327 62.2236Z" fill="white"/>
+<path d="M61.237 15.4957L53.9891 8.24781L11.327 50.9099C8.20281 54.0341 8.20281 59.0994 11.327 62.2236L29.0047 79.9013C32.1289 83.0255 37.1942 83.0255 40.3184 79.9013L82.9805 37.2392L75.7326 29.9913M61.237 15.4957L68.4848 8.24781L82.9805 22.7435L75.7326 29.9913M61.237 15.4957L75.7326 29.9913" stroke="#00297A" stroke-width="2" />
+<circle cx="46.0567" cy="44.7427" r="11" transform="rotate(-45 46.0567 44.7427)" stroke="#00297A" stroke-width="2"/>
+<mask id="path-5-outside-1_201_13130" maskUnits="userSpaceOnUse" x="-2.96281" y="32.7427" width="33" height="23" fill="black">
+<rect fill="white" x="-2.96281" y="32.7427" width="33" height="23"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M22.2759 45.3735L16.9067 50.7427L18.6767 52.5027L27.0567 44.1227L18.6767 35.7427L16.9067 37.5027L22.2776 42.8735H-0.962814V45.3735H22.2759Z"/>
+</mask>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M22.2759 45.3735L16.9067 50.7427L18.6767 52.5027L27.0567 44.1227L18.6767 35.7427L16.9067 37.5027L22.2776 42.8735H-0.962814V45.3735H22.2759Z" fill="#A7B5EC" class="siwFillSecondary"/>
+<path d="M16.9067 50.7427L15.4925 49.3285L14.0743 50.7467L15.4965 52.1609L16.9067 50.7427ZM22.2759 45.3735L23.6901 46.7877L27.1043 43.3735H22.2759V45.3735ZM18.6767 52.5027L17.2665 53.9209L18.6807 55.3271L20.0909 53.9169L18.6767 52.5027ZM27.0567 44.1227L28.4709 45.5369L29.8851 44.1227L28.4709 42.7085L27.0567 44.1227ZM18.6767 35.7427L20.0909 34.3285L18.6807 32.9182L17.2665 34.3245L18.6767 35.7427ZM16.9067 37.5027L15.4965 36.0845L14.0743 37.4987L15.4925 38.9169L16.9067 37.5027ZM22.2776 42.8735V44.8735H27.106L23.6918 41.4593L22.2776 42.8735ZM-0.962814 42.8735V40.8735H-2.96281V42.8735H-0.962814ZM-0.962814 45.3735H-2.96281V47.3735H-0.962814V45.3735ZM18.3209 52.1569L23.6901 46.7877L20.8616 43.9593L15.4925 49.3285L18.3209 52.1569ZM20.0869 51.0845L18.3169 49.3245L15.4965 52.1609L17.2665 53.9209L20.0869 51.0845ZM25.6425 42.7085L17.2625 51.0885L20.0909 53.9169L28.4709 45.5369L25.6425 42.7085ZM17.2625 37.1569L25.6425 45.5369L28.4709 42.7085L20.0909 34.3285L17.2625 37.1569ZM18.3169 38.9209L20.0869 37.1609L17.2665 34.3245L15.4965 36.0845L18.3169 38.9209ZM23.6918 41.4593L18.3209 36.0885L15.4925 38.9169L20.8634 44.2877L23.6918 41.4593ZM22.2776 40.8735H-0.962814V44.8735H22.2776V40.8735ZM-2.96281 42.8735V45.3735H1.03719V42.8735H-2.96281ZM-0.962814 47.3735H22.2759V43.3735H-0.962814V47.3735Z" fill="white" mask="url(#path-5-outside-1_201_13130)"/>
+</g>
+<defs>
+<clipPath id="clip0_201_13130">
+<rect width="91" height="85" rx="4" fill="white"/>
```

Comment:
Not sure this should be filled? @alexdahl-okta

Reply:
Good catch; should **not** be filled white.

I updated Figma, I think it should also be safe to just delete `fill="white"`